core/src/test/java/org/apache/iceberg/actions/TestSizeBasedRewriter.java

```diff
@@ -76,6 +76,27 @@ public void testSplitSizeLowerBound() {
     assertThat(splitSize).isLessThan(maxFileSize);
   }
 
+  @TestTemplate
+  public void testDeleteFileThresholdOption() {
+    SizeBasedDataFileRewriterImpl rewriter = new SizeBasedDataFileRewriterImpl(table);
+
+    Map<String, String> options = ImmutableMap.of(
+            SizeBasedDataRewriter.DELETE_FILE_THRESHOLD, "5"
```

Comment:
`DELETE_FILE_THRESHOLD` is different from the delete ratio and is for something else